examples/machine-files/IcelakeSP_Platinum-8360Y.yml

```diff
@@ -98,24 +98,23 @@ memory hierarchy:
     misses: 0  # this assumes LLC prefetcher to be off. may be measured with OFFCORE_RESPONSE_0_OPTIONS:PMC0:MATCH0=0x180
-    evicts: (CAS_COUNT_WR:MBOX0C[0-3] + CAS_COUNT_WR:MBOX1C[0-3] + CAS_COUNT_WR:MBOX2C[0-3]
-      + CAS_COUNT_WR:MBOX3C[0-3] + CAS_COUNT_WR:MBOX4C[0-3] + CAS_COUNT_WR:MBOX5C[0-3])
+    evicts: (CAS_COUNT_WR:MBOX0C[0-3] + CAS_COUNT_WR:MBOX1C[0-3] + CAS_COUNT_WR:MBOX2C[0-3] + CAS_COUNT_WR:MBOX3C[0-3] + CAS_COUNT_WR:MBOX4C[0-3] + CAS_COUNT_WR:MBOX5C[0-3] + CAS_COUNT_WR:MBOX6C[0-3] + CAS_COUNT_WR:MBOX7C[0-3])
   size per group: 27.00 MB
   threads per group: 18
 - level: MEM
   cores per group: 36
   upstream throughput: [full socket memory bandwidth, half-duplex]
   transfers overlap: false
   performance counter metrics:
-    loads: (CAS_COUNT_RD:MBOX0C[0-3] + CAS_COUNT_RD:MBOX1C[0-3] + CAS_COUNT_RD:MBOX2C[0-3] + CAS_COUNT_RD:MBOX3C[0-3] + CAS_COUNT_RD:MBOX4C[0-3] + CAS_COUNT_RD:MBOX5C[0-3])
-    stores: (CAS_COUNT_WR:MBOX0C[0-3] + CAS_COUNT_WR:MBOX1C[0-3] + CAS_COUNT_WR:MBOX2C[0-3] + CAS_COUNT_WR:MBOX3C[0-3] + CAS_COUNT_WR:MBOX4C[0-3] + CAS_COUNT_WR:MBOX5C[0-3])
+    loads: (CAS_COUNT_RD:MBOX0C[0-3] + CAS_COUNT_RD:MBOX1C[0-3] + CAS_COUNT_RD:MBOX2C[0-3] + CAS_COUNT_RD:MBOX3C[0-3] + CAS_COUNT_RD:MBOX4C[0-3] + CAS_COUNT_RD:MBOX5C[0-3] + CAS_COUNT_RD:MBOX6C[0-3] + CAS_COUNT_RD:MBOX7C[0-3])
+    stores: (CAS_COUNT_RD:MBOX0C[0-3] + CAS_COUNT_RD:MBOX1C[0-3] + CAS_COUNT_RD:MBOX2C[0-3] + CAS_COUNT_RD:MBOX3C[0-3] + CAS_COUNT_RD:MBOX4C[0-3] + CAS_COUNT_RD:MBOX5C[0-3] + CAS_COUNT_RD:MBOX6C[0-3] + CAS_COUNT_RD:MBOX7C[0-3])
```

Comment:
Line 117: Those should be 8 times `CAS_COUNT_WR`

Reply:
Updated